packages/location-field/src/index.js

```diff
@@ -74,7 +75,8 @@ class LocationField extends Component {
       menuVisible: false,
       message: null,
       geocodedFeatures: [],
-      activeIndex: null
+      activeIndex: null,
+      listBoxId: `listbox-${uniqueId()}`
```

Comment:
There is already an `optionKey` variable which is being used to set a unique key for react. I would suggest either re-using that for the aria-id if possible, or replacing that variable with the new listBoxId.

Reply:
I switched it up to use the optionKey. It works fine as can be seen in storybook and the snapshot tests. Always better to not introduce another dependency.